build.ps1

```diff
@@ -113,10 +113,9 @@ Invoke-BuildStep 'Cleaning artifacts' {
     -skip:($Fast -or $SkipXProj) `
     -ev +BuildErrors
 
-Invoke-BuildStep 'Enabling delay-signing' {
-        Enable-DelaySigning $MSPFXPath $NuGetPFXPath
+Invoke-BuildStep 'Configure delay signing' {
+        Configure-DelaySigning -MSPFXPath:$MSPFXPath -NuGetPFXPath:$NuGetPFXPath -CI:$CI
```

Comment:
I think it's required in this case.  If `$MSPFXPath` or `$NuGetPFXPath` isn't set, PowerShell reports an error:

    A positional parameter cannot be found that accepts argument 'False'.